Multisig/UI/Transactions/TransactionStatusView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  TransactionStatusView.swift
+//  Multisig
+//
+//  Created by Moaaz on 6/15/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct TransactionStatusView: View {
+    let status: TransactionStatus
+    var style: Style = .body
+    var body: some View {
+        HStack (spacing: 6) {
+            if [TransactionStatus.waitingConfirmation, TransactionStatus.waitingExecution].contains(status) {
```

Comment:
this seems to be repeating, time to extract it into a computed var in the "status" - to the "isPending" variable

Reply:
or use "isQueued"

Reply:
or "isWaiting"